astronomer/providers/amazon/aws/extractors/redshift.py

```diff
@@ -0,0 +1,235 @@
+import logging
+import traceback
+from typing import Any, Dict, List, Optional
+
+import attr
+from airflow.models.taskinstance import TaskInstance
+from openlineage.airflow.extractors.base import BaseExtractor, TaskMetadata
+from openlineage.airflow.utils import get_job_name
+from openlineage.client.facet import (
+    BaseFacet,
+    OutputStatisticsOutputDatasetFacet,
+    SqlJobFacet,
+)
+from openlineage.common.dataset import Dataset, Source
+from openlineage.common.models import DbColumn, DbTableSchema
+from openlineage.common.sql import DbTableMeta, SqlMeta, parse
+
+SCHEMA_URI = "https://raw.githubusercontent.com/OpenLineage/OpenLineage/main/spec/OpenLineage.json"
+
+
+class RedshiftAsyncExtractor(BaseExtractor):
```

Comment:
which RedshiftAsyncExtractor ? is it SQL or Cluster or Redshift Data, can add proper name for this extractor class?

Reply:
It is for Redshift Data and SQL so gave a generic name.